### PR TITLE
[13.0][MIG] website_sale_stock_provisioning_date: Migration to 13.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT27882

Normal migration.

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]
:+1: Tested

File: website_sale_stock_provisioning_date/models/product_product.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    def _get_next_provisioning_date(self):
+        domain = [
+            ("company_id", "=", self.env.user.company_id.id),
```
Comment (sergio-teruel) [inline website_sale_stock_provisioning_date/models/product_product.py]: you can use self.env.company directly in v13.
I have a doubt. Should the company be user's company or company of website??
What do you think?

**Review by sergio-teruel — CHANGES_REQUESTED** [review]
Thanks @ernestotejeda 
Only a little comment - question...

File: website_sale_stock_provisioning_date/models/product_product.py (hunk 1 above)
Comment (ernestotejeda) [reply website_sale_stock_provisioning_date/models/product_product.py]: I think it should be the website company to show the closest date on which the company corresponding to the website will receive the product. I'm going to change that condition of the domain then. You agree with me?

Comment (sergio-teruel) [reply website_sale_stock_provisioning_date/models/product_product.py]: perfect

**Comment by ernestotejeda** [comment]
@sergio-teruel Changes done

**Review by victoralmau — APPROVED** [review]
Code review

**Review by HaraldPanten — APPROVED** [review]

**Review by sergio-teruel — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Isn't this feature native in v14 and the ROADMAP should reflect that is not needed in v14?

**Comment by pedrobaeza** [comment]
@victoralmau did you check if the feature is native in v14?

**Comment by pedrobaeza** [comment]
Merging and we can add the comment later:

/ocabot merge nobump

**Pull request merged**